pkgs/development/tools/wiggle/default.nix

```diff
@@ -1,12 +1,15 @@
-{ lib, stdenv, fetchurl, ncurses, groff }:
```

Comment:
I already fixed this package in https://github.com/NixOS/nixpkgs/commit/2b6b345900f470dde8bdbeea24fddf9ff37749ed. It would be great if you could drop the changes from this PR to prevent a merge conflict.